config/packages/security.yaml

```diff
@@ -39,4 +39,5 @@ security:
     access_control:
         - { path: ^/login, roles: IS_AUTHENTICATED_ANONYMOUSLY }
         - { path: ^/register, roles: IS_AUTHENTICATED_ANONYMOUSLY }
-        - { path: ^/, roles: [IS_AUTHENTICATED_FULLY, ROLE_ADMIN] }
\ No newline at end of file
+        #- { path: ^/category/new, roles: [ROLE_ADMIN] }
+        - { path: ^/, roles: [IS_AUTHENTICATED_FULLY, ROLE_USER] }
```

Comment:
@DurandSacha just `ROLE_USER` is enough no ?

Reply:
it's Right, thanks